test/xpu_api/random/interface_tests/engines_methods.pass.cpp

```diff
@@ -317,8 +314,6 @@ test<oneapi::dpl::ranlux24>
 public:
     bool run(sycl::queue& queue)
     {
-        using result_type = typename oneapi::dpl::ranlux24::scalar_type;
```

Comment:
The same for that - it is a simple check of API.

Reply:
Fixed